services/app-web/src/components/ProgramSelect/ProgramSelect.tsx

```diff
@@ -50,8 +52,8 @@ export const ProgramSelect = ({
             })}
             className={styles.multiSelect}
             classNamePrefix="program-select"
-            id="programIDs"
-            name="programIDs"
+            id="programSelect"
```

Comment:
Wouldn't this be a bit problematic when we have multiple program selectboxes on the same page (such as in the multi-rate world which is coming soon). We would have multiple elements with the same html id on the page.  I am now thinking we should never be hard-coding this id at all fully - maybe make its value something like `${name}-select`  where name is always assigned to something specific like "contractPrograms" or "rate1programs"

Reply:
Doh, good catch.